docs/src/_getting-started/part3.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Part 3 - Inline Transformations, E(t)L
```

Comment:
@tayloramurphy - We've been very careful to avoid 'transforms' and the EtLT because it doesn't translate well and _appears_ on the surface to conflict with our core best practices. I don't want to use the word "transform" or "transformation" anywhere in the header here. I definitely would want to avoid the "little 't'" terminology because it then makes subsequent conversations about "the transformation layer" subject to the constant need to clarify: "which one?".

Even though mappers are very cool, my preference would be to keep mappers out of this initial tutorial and maybe move to an 'advanced topics' section. The other factor here is that we already have a large-enough challenge in getting users (from a variety of backgrounds) onboarded onto "what is ELT" and "how do I use Meltano for ELT". I think as an advanced topic, we'd have more time to let this breath.


Happy to chat more about this in a sidebar if helpful.

Reply:
On thinking about this a bit more...

The primary use case in the industry for `mappers` and `mappings` is data obfuscation and data protection - meaning: we actively want the data and don't want the data at the same time. Replicating the data in clear-text is a no-go for security reasons (or at least prohibitively expensive in terms of our liability), but not having the data at all is debilitating to our ability to get insights.

With this framing, if we simply swap the order of `part-3` and `part-4`, one path we could take is to say:

> In step 1 we intentionally did not import email addresses due to privacy considerations. Now we will use the "stream maps" advance feature to import only an anonymized one-way hash of the email address. This will allow downstream analytics applications to calculate "how many unique emails are there?" without ever importing those emails into our database.

Then the section header might be focused on "advanced data protection" or similar, as a specific use case for stream maps.

Reply:
@aaronsteers I'm in favor of the following:

* Swapping parts 3 and 4. By getting to the true dbt transformation first you run the full pipeline and complete the ELT
* Keeping Part 4 but not labeling it advanced. Stream maps are a key feature that separates us from other tools and should be highlighted and promoted
* Renaming the title to something akin to "Inline Data Mapping". This moves us away from the transformation language in the header. IN the body we talk about this type of transformation and how it's distinguished from dbt-style transformation
* Data protection is arguably the main use for mappers but they're powerful and can be used for other things as well. The framing I like is that it gives the user a lot of control over how the data lands. Data protection is one application of that
* We should figure out how to test and talk about E(t)LT. as much as I like EMLT or E(m)LT, nobody knows what we're talking about and at least E(t)LT gets people to understand that it's similar to ETL but clearly something is different!

Reply:
@aaronsteers @tayloramurphy 

- I definitely favor keeping it in the tutorial, have no strong opinion on whether 3 or 4 (fwiw, I chose 3 because it seemed to be an easier progression in terms of difficulty of the tutorial, because it's still inside the meltano context mostly, unlike dbt which leaves that context and requires the user to potentially also learn about dbt).
- I think this is way more than a nice to have. The core to solving data integration problems are the connectors. Thus they come in a generic form, which means we will always need to have the possibility to manipulate the output afterwards in some slight way. Same goes for all the big APIs of SaaS solutions. They are never tailored to our specific request, so having mappers is a pretty powerful tool here. 
- I would also favor the word "Inline Data Mapping" over the (one) use case of "Data protection", but thats mostly because in the customer interviews someone provided a good use case which simply was different (I think they added an identifier to the data). And quite a few others (related to the above point) might also go into a different direction.

I like the E(t)LT acronym because it's real easy to explain: We went from ETL to ELT because ELT gives us 10x more benefits. Turns out, small transformations, we call them more appropriately "inline data mappings" are very useful, so we added them back in (in a way that encourages small transformations, and discourages large ones).

Reply:
> I like the E(t)LT acronym because it's real easy to explain

I like it for that as well too. EtLT or E(t)LT renders nicely, connects to what people knows but is different enough to invite conversation.